File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int> out(nums.size());
        for (int i = 0; i < nums.size(); i++){
            int count = 0;
            for (int j = 0; j < nums.size(); j++){
                if (i != j and nums[j] < nums[i]){
                    count++;
                }
            }
            out[i] = count;
        }
        return out;
    }
};

File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        n = len(nums)
        out = [None] * n
        for i in range(n):
            count = 0
            for j in range(n):
                if (i != j and nums[j] < nums[i]):
                    count += 1
            out[i] = count
        return out
